data/themes/darktable.css

```diff
@@ -1539,8 +1539,9 @@ margin: 0 2px;
 {
   min-height: 0.95em;
   min-width: 0.95em;
-  margin: 0 0.75em;
+  margin: 0 0.45em;
```

Comment:
It's now me who find toggle button too small. 0.5em would be better.
Set `margin: 0 0.5em;`

Reply:
This makes my buttons smaller than in master when I use a smaller font. I mostly got to 0.45 by trial and error comparison with master.

Reply:
I think the extra padding is taking space from the height. If I use 0.5em padding and resize the icons to min 1.15em it looks the same as master.